app/views/posts/index.html.erb

```diff
@@ -1 +1,39 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<section class='main_section'>
+  <div class='user_card'>
+    <div class='user_img'>
+      <img src= <%= @user.photo %> alt= <%= @user.name %> />
+    </div>
+
+    <div class='user_stuff'>
+      <p class='user_name'><%= @user.name %></p>
+      <p class='posts_no'>Number of Posts: <%= @user.postscounter || 0 %></p>
+    </div>
+  </div>
+
+  <ul class='posts_list'>
+    <% @user.posts.each_with_index do |post, index|%>
+      <li>
+        <div class='post_card'>
+          <h4 class='post_header'>post #<%= index + 1%></h4>
```

Comment:
- [ ] The post title should be a link that allows see the comments of the post, please 🙏🏽 fix that issue to see the comments 👇🏽 

```ruby
<h4 class='post_header'><%= link_to post.title, user_posts_path(post) %></h4>
```

Reply:
Done